x/ref/runtime/internal/rt/runtime.go

```diff
@@ -485,12 +478,21 @@ func (r *Runtime) commonServerInit(ctx *context.T, opts ...rpc.ServerOpt) (*pubs
 	return id.settingsPublisher, otherOpts, nil
 }
 
+func (r *Runtime) warnIfPermissionsFlagsIgnored(ctx *context.T, auth security.Authorizer) {
+	if r.GetPermissionsSpec(ctx).Specified {
+		if auth == nil || reflect.TypeOf(auth).Elem().PkgPath() != "v.io/v23/security/access" {
```

Comment:
Might be worth noting somewhere that this will break if we add something that doesn't obey the command line arguments to the `v.io/v23/security/access` package.

Reply:
done.